signl4/README.md

```diff
@@ -81,11 +158,15 @@ SIGNL4 triggered and resolved events appear in your SIGNL4 app and web portal.
 The SIGNL4 integration does not include any service checks.
 
 ## Troubleshooting
-Need help? Contact [SIGNL4 Support][5].
+Need help? Contact [SIGNL4 Support][8].
 
 [1]: https://www.signl4.com
 [2]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/signl4/images/signl4-phone.png
-[3]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/signl4/images/datadog-webhook.png
-[4]: https://www.signl4.com/blog/portfolio_item/datadog_mobile_alerting/
-[5]: mailto:success@signl4.com
-[6]: https://app.datadoghq.com/account/settings#integrations/webhooks
+[3]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/signl4/images/datadog-alerts-webhook.png
+[4]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/signl4/images/datadog-incidents-webhook.png
+[5]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/signl4/images/datadog-incidents-rule.png
+[6]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/signl4/images/signl4-datadog-connector-app.png
```

Comment:
These links 404 for me.

Reply:
Thanks a lot for your comments and changes @urseberry. The missing files are here:
https://github.com/rons4/integrations-extras/tree/master/signl4/images
It seems I cannot get them into this PR. Probably my fault. Should I create a separate PR?

Reply:
Yes, if you are having trouble getting the images into this PR, you can create a separate one.

Reply:
If you are including them in a separate PR, can you get rid of these links for now? The validation is failing because these files don't exist.